### PR TITLE
Updated version numbers for OCP 4.5

### PR DESCRIPTION
**Pull request by ahardin-rh**
https://issues.redhat.com/browse/OSDOCS-1046

### CODE REVIEW

File: modules/cluster-logging-deploy-cli.adoc [hunk 1]
```diff
@@ -156,14 +156,14 @@ The Elasticsearch Operator is installed to the `openshift-operators-redhat` Name
...
-kube-node-lease                                         elasticsearch-operator.4.4.0-202004222248    Elasticsearch Operator   4.4.0-202004222248               Succeeded
-kube-public                                             elasticsearch-operator.4.4.0-202004222248    Elasticsearch Operator   4.4.0-202004222248               Succeeded
-kube-system                                             elasticsearch-operator.4.4.0-202004222248    Elasticsearch Operator   4.4.0-202004222248               Succeeded
-openshift-apiserver-operator                            elasticsearch-operator.4.4.0-202004222248    Elasticsearch Operator   4.4.0-202004222248               Succeeded
-openshift-apiserver                                     elasticsearch-operator.4.4.0-202004222248    Elasticsearch Operator   4.4.0-202004222248               Succeeded
-openshift-authentication-operator                       elasticsearch-operator.4.4.0-202004222248    Elasticsearch Operator   4.4.0-202004222248               Succeeded
-openshift-authentication                                elasticsearch-operator.4.4.0-202004222248    Elasticsearch Operator   4.4.0-202004222248               Succeeded
```
Comment (ahardin-rh) [inline modules/cluster-logging-deploy-cli.adoc:166]: I'm making a note to re-check these version numbers at 4.5 release time to ensure this is final.

**Comment by ahardin-rh** [comment]
@openshift/team-documentation Please review :heart:

**Comment by bergerhoffer** [comment]
LGTM as far as I can tell

**Review by lamek — COMMENTED** [review]
LGTM

**Comment by bobfuru** [comment]
LGTM, too

File: modules/rhel-compute-updating.adoc [hunk 2]
```diff
@@ -41,9 +41,9 @@ You must not enable firewalld later. If you do, you cannot access {product-title
 +
 ----
 # subscription-manager repos --disable=rhel-7-server-ansible-2.7-rpms  \
```
Comment (kalexand-rh) [inline modules/rhel-compute-updating.adoc:43]: @mtnbikenc, for a 4.4 to 4.5 upgrade, should we remove the step to swap the Ansible repo?

Comment (mtnbikenc) [reply modules/rhel-compute-updating.adoc:43]: It looks like we missed the requirement to upgrade to Ansible 2.9 for 4.4, so we should ensure 2.9 is enabled for 4.5.

4.3 requires [Ansible 2.8](https://github.com/openshift/openshift-ansible/blob/release-4.3/openshift-ansible.spec#L20)
4.4 requires [Ansible 2.9](https://github.com/openshift/openshift-ansible/blob/release-4.4/openshift-ansible.spec#L20)
4.5 requires [Ansible 2.9](https://github.com/openshift/openshift-ansible/blob/release-4.5/openshift-ansible.spec#L20)

**Comment by mburke5678** [comment]
@ahardin-rh LGTM

**Comment by mtnbikenc** [comment]
Ansible 2.9 update LGTM

**Comment by ahardin-rh** [comment]
Thank you, all! :rocket:

**Pull request merged**